### DISCUSSION TITLE
TRC20 network or solana USDT

### DISCUSSION THREAD

**Discussion by jomagalo** [btcpayserver/btcpayserver#3524] [General]
Do you plan to include some of the economic networks for USDT type TRC20 or solana?
Or is it possible to integrate it in some way?

Best regards

**Comment by sunchous** [comment]
I didn't find any docs about possibility of integration USDT(trc20) and btcpayserver...
But this token becomes more and more popular every day.

**Reply by samuelg78** [reply]
Any update for the plugin?

**Comment by jomagalo** [comment]
and surely they will not integrate it, because they have a plugin for it that charges commissions for the service.

**Reply by sunchous** [reply]
could you send me the link to this plugin, please?

**Comment by Kukks** [comment]
https://docs.btcpayserver.org/FAQ/Altcoin/#can-an-xyz-coin-be-added-in-btcpay
> Bitcoin is the only focus of the project and its core developers.

BTCPay developers don't add alternative coins on request. Adding a new coin explicitly depends on the community and developers of those coins. Furthermore, BTCPay developers do not spend excessive time testing nor maintaining the altcoins. If you're submitting a PR for a new coin, make sure that it works. If the altcoin integration is not actively maintained it will be removed from BTCPay.

**Comment by nosovk** [comment]
TRC20 now supported  https://github.com/btcpayserver-tether/BTCPayServer.Plugins.USDt
hope solana will also come some day
